examples/custom-node/src/primitives/extended_op_tx_envelope.rs

```diff
@@ -26,11 +26,13 @@ macro_rules! delegate {
 }
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize, Hash, Eq, PartialEq)]
-pub enum ExtendedOpTxEnvelope<T> {
-    BuiltIn(Box<OpTxEnvelope>),
-    Other(T),
+pub enum ExtendedTxEnvelope<BuiltIn, Other> {
+    BuiltIn(Box<BuiltIn>),
```

Comment:
imo we can remove the box here because we expect that both variant have more or less the same size